packages/amplify-go-function-runtime-provider/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "strict": true,
     "outDir": "./lib",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "composite": true,
```

Comment:
composite is set in the root tsconfig

Reply:
ahh you made me make this mistake :D :D removing. just removed strict which is also set.

Reply:
fixed.